### PR TITLE
Improvements to hypothesis tab

### LINKED ISSUE #371: [FEATURE REQUEST] Improvements to hypothesis tab
* Rename the "hypothesis" tab "optimiser"
* Add a checkbox in the "hypothesis" tab to enable/disable optimisation. This flag would determine whether  the `tracker.optimize()` method is run here:
https://github.com/quantumjot/btrack/blob/003407e00569e3718aceacc849036835f477a121/btrack/napari/main.py#L298-L301

### PR DESCRIPTION
**Pull request by paddyroddy**
This PR is a catch all for various changes related to the napari plugin and things done in the last few days. I'll make a separate issue to test this.

image

### CODE REVIEW

**Comment by paddyroddy** [comment]
Fixes #381

**Review by quantumjot — APPROVED** [review]
Looks good. I think everything is working

**Comment by paddyroddy** [comment]
> it's looking good! You can add a callback with the [other ones](https://github.com/quantumjot/btrack/blob/2ec77286676c5fb65458f7d4fa59beb089906a2e/btrack/napari/main.py#L64-L97) to disable the Optimiser tab when `enable optimisation` is unchecked:
> 
> ```python
> btrack_widget.enable_optimisation.toggled.connect(
>     lambda is_checked: btrack_widget._tabs.setTabEnabled(2, is_checked)
> )
> ```

Separate PR

**Review by p-j-smith — APPROVED** [review]

File: btrack/napari/main.py [hunk 1]
```diff
@@ -80,6 +80,14 @@ def create_btrack_widget() -> btrack.napari.widgets.BtrackWidget:
         lambda selected: select_config(btrack_widget, all_configs, selected),
     )
 
+    # Disable the Optimiser tab if unchecked
+    for tab in range(btrack_widget._tabs.count()):
+        if btrack_widget._tabs.tabText(tab) == "Optimiser":
+            break
+    btrack_widget.enable_optimisation.toggled.connect(
+        lambda is_checked: btrack_widget._tabs.setTabEnabled(tab, is_checked)
+    )
```
Comment (paddyroddy) [inline btrack/napari/main.py:89]: This was the best way I could get it without relying on index @p-j-smith

**Comment by paddyroddy** [comment]
Now looks like this. Decided to also shrink the logo and move `Track` above the config so it is always visible. Will now write some tests #388 
image

File: btrack/napari/main.py (hunk 1 above)
Comment (p-j-smith) [reply btrack/napari/main.py:89]: @paddyroddy I think that's an okay way to do it. Or if you prefer you can get the index of one of the tabs using the [widget we put in that tab](https://github.com/quantumjot/btrack/blob/e2ee8d82713496655be915d7659dcde82a3de090/btrack/napari/widgets/create_ui.py#L137-L139). As we're not storing the widget itself, you could use one of the child widgets (e.g. [`hypotheses`](https://github.com/quantumjot/btrack/blob/e2ee8d82713496655be915d7659dcde82a3de090/btrack/napari/widgets/_optimiser.py#L34)) and find its parent:

```python
optimiser_tab_index = btrack_widget._tabs.indexOf(btrack_widget.hypotheses.parentWidget())
```

Comment (paddyroddy) [reply btrack/napari/main.py:89]: Yeah, I saw that, but I'm not convinced it's any better! Especially as some widgets have moved tabs

**Pull request merged**